CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to
 - [#4926](https://github.com/firecracker-microvm/firecracker/pull/4926): Replace
   underlying implementation for seccompiler from in house one in favor of
   `libseccomp` which produces smaller and more optimized BPF code.
+- [#5021](https://github.com/firecracker-microvm/firecracker/pull/5021) Fix UFFD
```

Comment:
Nit: This reads a bit like "we fixed UFFD handlers". We can rephrase it to make it clear we fixed Firecracker behaviour that led to that. We can also start with "If a balloon device is used" which will make some readers stop reading.

Reply:
changed!